### DISCUSSION TITLE
Deleting files increases the backup size

### DISCUSSION THREAD

**Discussion by vidplayzz** [MaxiHuHe04/iTunes-Backup-Explorer#70] [Q&A]
I have an issue where I backed up my iphone 15 via itunes and I cannot restore my phone with the backup because the backup file is too large(?). Some guy said that he had the same issue and he used this program to reduce his backup size so he restored his phone succesfully. However, when I delete a file, the backup folder increases in size. I'm going crazy because i have lost every file in the last 3 years and i hope i will manage to restore my phone by reducing the backup size. The folder went from 157gb to 164gb (my phone has 128gb of storage). Can somebody help me in any possible way so i manage to restore my phone.

**Comment by MaxiHuHe04** [comment]
The size of the backup folder slightly increases when you delete a file because the program only moves the deleted files into a backup sub-folder and backs up metadata belonging to them. However, the size of the backup itself that iTunes uses to restore the phone does decrease when you delete something.
There is one caveat. In the current version, after you deleted some specific files or a specific amount of files, it can sometimes happen that iTunes claims that the backup is corrupted.
I can't promise anything, but I plan on taking a closer look at this issue this or next week. If your data is important, I would wait a bit with deleting more files. Trying to restore it now should not make it worse however in case you just haven't tried yet because of the folder size.

**Reply by vidplayzz** [reply]
Thank you so much man. I have over 5 years of photos there and i srsly wouldnt know what to do if i lost everything. I dont care abt anything other than my camera roll. Id be very thankful if you manage to help me in any way. I just want my camera roll back. Thank you so much for looking into it if you do.

**Reply by don-pulubi** [reply]
@MaxiHuHe04  where you able to figure out the "corrupted" issue? it happened to me recently and good thing I was able to backup my backup ;)

**Reply by MaxiHuHe04** [reply]
Hi @vidplayzz @don-pulubi, it took a bit longer than I had hoped, but the backup corruption issues should finally be fixed now in version 1.6.
@don-pulubi I would be happy to hear whether the [instructions](https://github.com/MaxiHuHe04/iTunes-Backup-Explorer/releases/tag/v1.6) for making such backups recoverable again work for you in case you still have the one that had been reported as corrupted.
